README.md

```diff
@@ -64,17 +76,17 @@ deblur workflow --help
 
 Positive and Negative Filtering
 ===============================
-By default, deblur uses positive filtering, keeping only 16S sequences (based on homology to [Greengenes](http://greengenes.secondgenome.com/) [88% representative set](deblur/support_files/88_otus.fasta)). For example:
+Deblur can use two types of filtering on the sequences:
 
-```
-deblur workflow --seqs-fp all_samples.fna --output-dir output
-```
+- negative mode - removes [known artifact sequences](deblur/support_files/artifacts.fa) (i.e. PhiX and Adapter sequences).
 
-Negative filtering can be selected using the '-n' flag. This causes deblurring to keep all sequences except for [known artifact sequences](deblur/support_files/artifacts.fa) (i.e. PhiX and Adapter sequences), so other non-16S sequences are retained. For example:
+- positive mode - keeps only sequences similar to [known 16S sequences](deblur/support_files/88_otus.fasta) (GreenGenes 88%)
 
-```
-deblur workflow --seqs-fp all_samples.fna --output-dir output -n
-```
+By default, deblur uses negative mode filtering to remove known artifact (i.e. PhiX and Adapter sequences) prior to denoising. The output of deblur contains two files: final.biom, which includes all sOTUs, and final.only16s.biom, which contains the output of positive filtering of the sOTUs (only sOTUs similar to 16S sequences).
```

Comment:
i agree. we can use new names. maybe all.biom and positive-filtered.biom ?
what do other people think